ArduPlane/quadplane.cpp

```diff
@@ -2895,6 +2921,39 @@ void QuadPlane::vtol_position_controller(void)
     }
 }
 
+void QuadPlane::assign_tilt_to_fwd_thr(void) {
+#if QAUTOTUNE_ENABLED
+    if (plane.control_mode == &plane.mode_qautotune) {
```

Comment:
I don't see any code path that would call this function in qautotune? Could put a SITL only panic in to ensure that remains the case.

Reply:
I've checked the code twice and can't find a way that this function can be called in QAUTOTUNE so will remove it.